src/main/java/io/github/darealturtywurty/turtychemistry/client/renderer/MolderBlockEntityRenderer.java

```diff
@@ -0,0 +1,55 @@
+package io.github.darealturtywurty.turtychemistry.client.renderer;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.math.Vector3f;
+import io.github.darealturtywurty.turtychemistry.common.block.entity.MolderBlockEntity;
+import io.github.darealturtywurty.turtychemistry.common.block.molder.MolderBlock;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.client.renderer.block.model.ItemTransforms;
+import net.minecraft.client.renderer.blockentity.BlockEntityRenderer;
+import net.minecraft.client.renderer.blockentity.BlockEntityRendererProvider;
+import net.minecraft.client.renderer.entity.ItemRenderer;
+import org.jetbrains.annotations.NotNull;
+
+public final class MolderBlockEntityRenderer implements BlockEntityRenderer<MolderBlockEntity> {
+
+
+    private final ItemRenderer itemRenderer;
+
+
+    public MolderBlockEntityRenderer(BlockEntityRendererProvider.Context ctx) {
+        this.itemRenderer = ctx.getItemRenderer();
```

Comment:
```suggestion
public final class MolderBlockEntityRenderer implements BlockEntityRenderer<MolderBlockEntity> {
    private final ItemRenderer itemRenderer;

    public MolderBlockEntityRenderer(BlockEntityRendererProvider.Context ctx) {
        this.itemRenderer = ctx.getItemRenderer();
```

Reply:
well I think to finish this PR it requires art assets for the molder screen to be made (also Intellij still wont format it correctly and sometimes it thinks it's a good idea to un-format some stuff)